blog/v4_announcement.ipynb

```diff
@@ -8,47 +8,30 @@
     "(v4_announcement)=\n",
     "\n",
     "# PyMC 4.0 Release Announcement\n",
-    "\n",
-    ":::{post} Jun 10, 2022\n",
+    ":::{post} June 6, 2022\n",
     ":tags: release, aesara, jax\n",
     ":category: news\n",
     ":author: Thomas Wiecki\n",
-    ":::\n",
```

Comment:
This seems to be deleting the closing colon fence which should be there, I think sphinx is clever enough to close it and render everything right but better not tempt our luck